### PR TITLE
get smallest

### PR DESCRIPTION
**Pull request by basharreda**

### CODE REVIEW

File: GitAssignment/src/com/company/Main.java [hunk 1]
```diff
@@ -1,8 +1,39 @@
 package com.company;
 
+import java.util.Scanner;
+
 public class Main {
 
+    public static void getSmallest()
+    {
+                System.out.println("Get Smallest");
```
Comment (HossamHoussien) [inline GitAssignment/src/com/company/Main.java]: Good work,
I think you should make the function independent from input data. I.e. the main function will handle the input for all functions. Just make your function takes an array as a parameter and do your logic then return the results.

**Review by Bassant-Mohamed — COMMENTED** [review]
the input "array" is taken in the main not inside your function

**Review by karimhosam — APPROVED** [review]

**Review by Bassant-Mohamed — COMMENTED** [review]
the function need the smallest "prime" number not the smallest number

**Review by DiinaSaeed — APPROVED** [review]

**Comment by AhmedMaherAli** [comment]
Hi Bashar, i got your idea here but it can't handel the numbers greater than 9*9=81 for example if your array contains those two numbers arr={121,169}; your code here will deal with both of them as primes and your output will be 121, but unfortunately both of them are not primes as 121 is divisible by 11 and 169 is divisible by 13. 

Hint: you can use Sieve function to generate the prime numbers before your algorithm to find the smallest.
another solution is to try (enk trf3 elcode zy ma hwa keda w t7awel tklft el mo3ed ellhyna24lk we mt5leho4 yd5l arkam akbr mn 81) :""""D

**Comment by AhmedMaherAli** [comment]
if you wanted to implement the first solution i can send you the sieve function.

**Review by Moataz987 — APPROVED** [review]

**Pull request merged**